service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java

```diff
@@ -24,7 +25,19 @@ public static String formatJudgeTitleAndName(JudgeAndLegalAdvisor judgeAndLegalA
             .orElse("");
     }
 
+    public static String formatJudgeTitleAndNameForDraftSDO(JudgeAndLegalAdvisor judgeAndLegalAdvisor,
+                                                            String emptyPlaceholder) {
+        return Optional.ofNullable(judgeAndLegalAdvisor)
+            .filter(judge -> judge.getJudgeTitle() != null)
+            .map(JudgeAndLegalAdvisorHelper::mapJudgeOrAdvisor)
+            .orElse(emptyPlaceholder);
```

Comment:
(0) Just a thought - this could return Optional.empty if not present and the client would need to format if required with blank.

(-1) Or/and - Above methods should be refactored to make use of this one (the one we are doing map + orElse "")

Reply:
decided not to make any changes to this at the moment due to tight deadline for this story